docs/topics/api/accounts.rst

```diff
@@ -73,6 +73,55 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 .. http:get:: /api/v3/accounts/profile/
 
 
+----
+Edit
+----
+
+.. _`account-edit`:
+
+.. note::
+    This API requires :doc:`authentication <auth>` and `Users:Edit`
+    permission to edit accounts other than your own.
+
+This endpoint allows some of the details for an account to be updated.  Any fields
+in the :ref:`account <account-object>` (or :ref:`self <account-object-self>`)
+but not listed below are not editable and will be ignored in the patch request.
+
+.. http:patch:: /api/v3/accounts/account/(int:user_id)/
+
+    .. _account-edit-request:
+
+    :<json string|null biography: More details about the user.  No links are allowed.
+    :<json string|null display_name: The name chosen by the user.
+    :<json string|null homepage: The user's website.
+    :<json string|null location: The location of the user.
+    :<json string|null occupation: The occupation of the user.
+    :<json string|null username: username to be used in the account url.  The username can only contain letters, numbers, underscores or hyphens. All-number usernames are prohibited as they conflict with user-ids.
+
+
+-------------------
+Uploading a picture
+-------------------
+
+To upload a picture for the profile the request must be sent as content-type `multipart/form-data` instead of JSON.
+Images must be either PNG or JPG and the maximum file size is 4MB.
+Other :ref:`editable values <account-edit-request>` can be set at the same time.
```

Comment:
Paging @EnTeQuAk for the discussion.

As mentioned on IRC I am not a big fan of doing `multipart/form-data` in APIs. For clients it means doing things in a completely different way just for this particular API, that's a bit weird. 

I'd prefer a separate endpoint that you can `POST`/`PUT` to the data directly with the right `image/<something>` content type. This can be done in DRF by declaring a `detail_route` on your viewset, passing `parser_classes = (FileUploadParser,)` and making sure that the clients are allowed to set `Content-Type` and `Content-Disposition` headers via CORS. For the curl example, you'd only need to update the URL and change `--form` to `--data-binary`, removing the `picture_upload=` part since the data is sent directly to the endpoint.